### DISCUSSION TITLE
google features are being removed from chromium

### DISCUSSION THREAD

**Discussion by FarisZR** [Eloston/ungoogled-chromium#1351] [General]
hi!, google announced very important changes to chromium which will be applied in march of this year

## the following features will be removed:
- Google account sync
- Geolocation
- Click to Call
- Chrome spelling API
- Contacts API
- Chrome translate element
- Safe browsing

what does this mean for ungoogled chromium ?, does this remove the need for ungoogled chromium entirely ?

source: https://www.omgubuntu.co.uk/2021/01/chromium-sync-google-api-removed

**Comment by megastallman** [comment]
No. Google is still not removing telemetry. So we still need to go Ungoogled. But I hope this cleanup makes Eloston's life easier, by reducing the patches count to support.

**Comment by Eloston** [comment]
The original Google blog post just states that they're "limiting access to our private Chrome APIs", which is most likely on the server-side. It's hard to say what that implies for the Chromium codebase; it'll probably take several major versions before we see any major changes.

**Comment by wchen342** [comment]
There seems to be no plan to remove UMA (User Metric Analysis) codes from what I see in v88. As mentioned above they are telemetries. They are supposed to be opt-in but these recordings widely exist in the codes so I cannot say for sure you can opt-out every one of them.
